src/components/async-resource/index.js

```diff
@@ -12,12 +12,13 @@ const AsyncResource = ({
   children,
   error,
   failedContent,
+  hasData,
```

Comment:
isn't this component used by several other components - a change to the contract might break other stuff?

Reply:
No, this prop is populated by the `mapStateToProps/connect()` wrapper, on line 80.